assets/telemeter-client/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: telemeter-client
+  name: telemeter-client
+  namespace: openshift-monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: telemeter-client
+  template:
+    metadata:
+      labels:
+        k8s-app: telemeter-client
+    spec:
+      containers:
+      - command:
+        - /usr/bin/telemeter-client
+        - --id=$(ID)
+        - --from=https://prometheus-k8s.openshift-monitoring.svc:9091
+        - --from-ca-file=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+        - --from-token-file=/var/run/secrets/kubernetes.io/serviceaccount/token
+        - --to=
```

Comment:
this should have a value no? (either here or in the code below, but I don't see it in either)

Reply:
this is what I was referring to in the PR description when I spoke about follow ups. We need to determine the production URL for the telemeter server; once we have that we'll configure it in the telemeter jsonnet and regenerate the manifests here. Does that sound good?

Reply:
Ack. My feeling is that this will need to be configurable in some way (maybe a flag on the cluster-monitoring-operator?).

Reply:
Yes this sounds completely reasonable. We need to be able to modify CMO deployments for testing, production, etc and be able to report to different telemeter servers. Either in a flag or as another field in the CMO configmap. I'll plumb the URL into a an environment variable

Reply:
ack :+1: